### PR TITLE
Completed CSS-Two-Column-Layout

### PR DESCRIPTION
**Pull request by Henry-Nichols**
image

### CODE REVIEW

File: css-two-column-layout/styles.css [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+.header {
+  text-align: center;
+}
+
+.info-container {
+  display: flex;
+}
+
+.info {
+  /* border: red solid; */
```
Comment (codingcodymiller) [inline css-two-column-layout/styles.css:58]: Please remove commented code prior to submitting

File: css-two-column-layout/styles.css [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+  display: flex;
+}
+
+.i {
+  /* border: red solid; */
```
Comment (codingcodymiller) [inline css-two-column-layout/styles.css:63]: Please remove commented code prior to submitting

File: css-two-column-layout/styles.css [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+
+.left-col {
+  display: flex;
+  flex-direction: column;
+}
+
+.intu-txt {
+  width: 60%;
+
+  /* border: 2px solid blue; */
```
Comment (codingcodymiller) [inline css-two-column-layout/styles.css:76]: Please remove commented code prior to submitting

**Review by codingcodymiller — CHANGES_REQUESTED** [review]
Fantastic work, man!! I have one minor adjustment for you:

**Review by codingcodymiller — APPROVED** [review]
Very nice!!

**Comment by Henry-Nichols** [comment]
Thank you Cody! I appreciate the feedback!

**Pull request merged**